util/update_bundled_ca_certificates.rb

```diff
@@ -4,6 +4,7 @@
 
 URIS = [
   URI('https://rubygems.org'),
+  URI('https://index.rubygems.org'),
   URI('https://staging.rubygems.org'),
   URI('https://s3.amazonaws.com'),
   URI('https://fastly.rubygems.org'),
```

Comment:
It's not used yet. It's currently pointed to staging but it's the same cert that production will use. My plan is to keep it active and pointing to Fastly even if we needed to change the root domain away from fastly for some reason. So yes, it's here to pull in the new cert that will be used soon.